### DISCUSSION TITLE
Translation pop-up problem

### DISCUSSION THREAD

**Discussion by nokuchi** [uetchy/Polyglot#133] [Q&A] [answered]
I've been using Polyglot for a while, love it immensely. However, very recently if I try to translate something in Safari, the translation box opens for only a second and then closes. It used to stay open so I could read it. I've tried auto translation, clicking the icon in the menu bar, clicking the icon to confirm, or using a key combination but the problem is the same. The translation box closes as soon as it opens with no time to read anything in it, making the extension unusable. Some help to fix this problem would be most appreciated.

**Comment by gingerbeardman** [comment, marked as answer]
See https://github.com/uetchy/Polyglot/issues/128#issuecomment-1487229870 for workaround.

This workaround is the best we can do right now.

To release a new version of the app/extension we need @uetchy but I have not heard from them in a long time.
